tests/test_main.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+import pytest
+
+from xtractmime.__init__ import (
+    _should_check_for_apache_bug,
+    _is_match_mime_pattern,
+    _read_resource_header,
+)
+
+from xtractmime.utils import _is_mp4_signature
```

Comment:
```suggestion
import unittest

import pytest

from xtractmime import (
    _should_check_for_apache_bug,
    _is_match_mime_pattern,
    _read_resource_header,
)
from xtractmime.utils import _is_mp4_signature
```

- https://www.python.org/dev/peps/pep-0008/#imports
- https://docs.python.org/3/tutorial/modules.html#packages